### PR TITLE
homework wee5 

### PR DESCRIPTION
**Pull request by Elias011**
I have fixed my the issues of homework week 4
I also tried to make it less confusing and I hope that the formatting is now  better than before

### CODE REVIEW

**Comment by isaachinman** [comment]
Now seeing this error:

screen shot 2018-01-28 at 20 52 35

Same issue as in #46.

**Comment by Elias011** [comment]
Sorry Isaac!! I forgot to --save 'Moment' in my package.json

File: todo-app/src/stores/TodoStore.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      .then(res => res.json())
+      .then(res => {
+        this.loadTodoItems(false)
+      })
+
+  }
+
+  @action
+
+  handleSaveEdit(id, description) {
```
Comment (isaachinman) [inline todo-app/src/stores/TodoStore.js]: First, the decorator should not have a line break after it.

Second, you have like 4-5 declarations of a `handleSaveEdit` function in this codebase, which is confusing.

Third, this is a MobX store, not a React component. You should not use the `handle` naming as this is not an event hander at all, it is a MobX action.

File: todo-app/src/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import TodoForm from './TodoForm';
+import loadinImage from './loadingImage.gif';
+
+@inject('todoStore')
+@observer
+
+export default class TodoList extends React.Component {
+
+  state = {
+    editingTodoID: null
```
Comment (isaachinman) [inline todo-app/src/TodoList.js]: Why is this state not inside your MobX store? Please move it inside your MobX store unless you can think of a good reason why it should not be there.

File: todo-app/src/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        <img src={loadinImage} alt='loading' />
+      )
+    }
+    return (
+      <div>
+        <TodoForm
+          ontodoAdd={this.handleTodoAdd}
+        />
+        <ul className='tasks'>
+          {todoStore.loadingTodos}
```
Comment (isaachinman) [inline todo-app/src/TodoList.js]: What is happening here? You can't render booleans.

File: todo-app/src/todo.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[
```
Comment (isaachinman) [inline todo-app/src/todo.json]: This file can be removed completely.

File: todo-app/src/storage.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export function loadTodoItems() {
```
Comment (isaachinman) [inline todo-app/src/storage.js]: This file can be removed completely.

**Pull request closed without merging**